pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1127,6 +1127,9 @@ public interface MissingProperSuffix extends javax.ejb.EJBLocalObject {}    // n
 A local variable assigned only once can be declared final.
         </description>
         <priority>3</priority>
+        <properties>
```

Comment:
For java-based rules such as this one, we don't need to declare the properties in XML - it's enough, if it is declared in the rule itself by calling `definePropertyDescriptor(...)`.

So, you can remove this change in codestyle.xml

Reply:
Done!